python/spec/dependabot/python/update_checker/index_finder_spec.rb

```diff
@@ -306,6 +320,43 @@
         end
       end
 
+      context "set in a pyproject.toml file and marked as explicit" do
```

Comment:
You'll normally start context with `when` or `with`. For example, `context "when set in a pyproject.toml file and marked as explicit" do`. If you do this, when you run `rspec --format documentation`, it reads nicer :)